src/artery/testbed/CMakeLists.txt

```diff
@@ -26,12 +27,29 @@ if(Protobuf_FOUND)
         testbed PUBLIC
         CubeConnection.cc
         OtaInterfaceCube.cc
+    )
+
+    protobuf_generate_cpp(
+        TESTBED_PROTO_SRCS TESTBED_PROTO_HDRS
         ${CUBE_PROTO_DIR}/nfiniity_cube_radio.proto
     )
 
-    protobuf_generate(TARGET testbed IMPORT_DIRS ${CUBE_PROTO_DIR})
-    target_include_directories(testbed PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-    target_link_libraries(testbed PUBLIC protobuf::libprotobuf)
```

Comment:
It should be okay to use protobuf_generate_cpp but still link to the protobuf::libprotobuf target.